packages/edit-post/src/components/visual-editor/style.scss

```diff
@@ -27,7 +27,7 @@
 
 .edit-post-visual-editor .block-editor-writing-flow__click-redirect {
 	// Allow the page to be scrolled with the last block in the middle.
```

Comment:
Just wanted to document with a screenshot, that this is still the case:

![Screenshot 2020-02-17 at 15 38 20](https://user-images.githubusercontent.com/1204802/74663160-903e3c80-519b-11ea-80b2-f4490c9b0c76.png)